src/webview/mod.rs

```diff
@@ -135,7 +153,7 @@ pub struct WebViewAttributes {
   /// [bug]: https://bugs.webkit.org/show_bug.cgi?id=229034
   pub custom_protocols: Vec<(
     String,
-    Box<dyn Fn(&Request<Vec<u8>>) -> Result<Response<Cow<'static, [u8]>>>>,
+    Box<dyn Fn(Request<Vec<u8>>, RequestAsyncResponder) -> Result<()>>,
```

Comment:
```suggestion
    Box<dyn Fn(Request<Vec<u8>>, RequestAsyncResponder)>,
```

Since the decision is removing Result, let's change it and remove the underlying function it calls.

Reply:
huh, we may need to add `custom_protocols_async` field here and keep the existing one as is to avoid a breaking change.

Reply:
let me know if it looks good now @wusyong